### ISSUE TITLE
java.lang.NoSuchMethodError: 'void org.bukkit.command.CommandSender.sendRichMessage(java.lang.String)'

### ISSUE THREAD

**Issue by kotazzz** [Konstanius/MineCICD#2]
```
[20:55:56 WARN]: [MineCICD] Plugin MineCICD v1.3 generated an exception while executing task 1362
java.lang.NoSuchMethodError: 'void org.bukkit.command.CommandSender.sendRichMessage(java.lang.String)'
        at ml.konstanius.minecicd.BaseCommand.lambda$onCommand$1(BaseCommand.java:794) ~[?:?]
        at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:100) ~[patched_1.16.5.jar:git-Purpur-1171]
        at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:58) ~[patched_1.16.5.jar:git-Purpur-1171]
        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.16.5.jar:git-Purpur-1171]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
        at java.lang.Thread.run(Thread.java:833) [?:?]
```
- Plugin: `MineCICD-1.3.jar`
- Server: purpur 1.16.5 (latest)
- Java: 17

I'm just starting to use the plugin. However, I can't get a response from the commands in the chat or in the console due to an error. Maybe the error only affects the text in the chat and the plugin works, but I won't see it

**Comment by Konstanius** [comment]
This plugin was developed for paper Spigot servers. If the method of sending rich messages using the mini message format is not given on other versions, i can't guarantee it to work.

However, I'm planning on releasing a compatibility version soon which makes the plugin able to work on Windows and bukkit as well, but that's gonna take some time.

**Comment by Konstanius** [comment]
As information for those interested into this issue / plugin, the plugin is currently back under active development and is being rewritten in its entirety.
All issues will be fixed and marked as closed with the new release, which improves stability, compatibility, performance and enables more features.

**Comment by Konstanius** [comment]
Fixed alongside MineCICD 2.0 Release
See https://github.com/Konstanius/MineCICD/commit/92dbd0e6daf093352b5790524a930f5ce87fbc22 for changes
